universal-application-tool-0.0.1/app/services/export/ExporterService.java

```diff
@@ -115,7 +116,7 @@ public String exportCsv(CsvExporter csvExporter, ImmutableList<Application> appl
   CsvExportConfig generateDefaultCsvConfig(long programId) {
     ImmutableList<Application> applications;
     try {
-      applications = programService.getProgramApplications(programId);
+      applications = programService.getSubmittedProgramApplications(programId);
```

Comment:
More a comment about the previous implementation, but now that we're doing some more work inside getSubmittedProgramApplications, would it make more sense to pass in the submitted applications from getProgramCsv rather than looking them up here again by programId?

Reply:
I did a small refactor on getProgramCsv so that getSubmittedProgramApplications will not be called multiple times unnecessarily